newswires/client/src/WireItemList.tsx

```diff
@@ -146,7 +147,7 @@ function MaybeSecondaryCardContent({
 		return <p>{subhead}</p>;
 	}
 	const maybeBodyTextPreview = bodyText
-		? sanitizeHtml(bodyText.replace(/(<br \/>|<\/p>)/g, '&nbsp;'), {
+		? sanitizeHtml(bodyText.replace(/(<br \/>|<\/p>|<\/td>)/g, ' '), {
```

Comment:
Curios about the nbsp change to a normal space here?

Reply:
I can't find a good reference for this, but my understanding is that the usual line-wrapping algorithm won't break a line on an `&nbsp;` in the way it does for regular spaces. I think the issue here could also be addressed with CSS (something like `word-break: break-all;`) but this felt more straightforward, and less likely to affect other kinds of subheadings?